src/main/java/teammates/storage/sqlentity/responses/FeedbackMissingResponse.java

```diff
@@ -0,0 +1,36 @@
+package teammates.storage.sqlentity.responses;
+
+import teammates.common.datatransfer.questions.FeedbackTextResponseDetails;
+import teammates.storage.sqlentity.FeedbackQuestion;
+
+/**
+ * Represents a missing response.
+ */
+public class FeedbackMissingResponse extends FeedbackTextResponse {
+
+    private String giverSectionName;
+    private String recipientSectionName;
+
+    protected FeedbackMissingResponse() {
+        // required by Hibernate
+    }
+
+    public FeedbackMissingResponse(
+            FeedbackQuestion feedbackQuestion, String giver,
+            String giverSectionName, String recipient, String recipientSectionName
+    ) {
+        super(feedbackQuestion, giver, null, recipient, null, new FeedbackTextResponseDetails("No Response"));
```

Comment:
Since "No Response" string literal is used multiple times, we can consider putting it into the Const file for it. This way, if we ever want to update the literal, we dont need to find all occurrences and update multiple lines and can just update the Const.